apps/ledger-live-mobile/src/components/RootNavigator/AccountsNavigator.tsx

```diff
@@ -75,6 +79,24 @@ export default function AccountsNavigator() {
           headerShown: false,
         }}
       />
+      {accountListUIFF?.enabled && (
+        <>
+          <Stack.Screen
+            name={ScreenName.AccountsList}
+            component={AccountsList}
+            options={{
+              headerTitle: "",
+            }}
+          />
+          <Stack.Screen
```

Comment:
as we can navigate to ScreenName.AssetsList via it's own navigator 
```typescript
navigate(NavigatorName.Asset, {screen: ScreenName.AssetsList ....})
```
, i think you can delete this